packages/reputation-miner/bin/forked.js

```diff
@@ -0,0 +1,106 @@
+const hre = require("hardhat");
```

Comment:
Any reason you don't use `globals` here?

Reply:
This isn't being run from inside hardhat, so I don't think it exists?